flask_resty/spec/spec_declaration.py

```diff
@@ -0,0 +1,9 @@
+class SpecDeclaration(object):
+
+    def update_spec(self, path, spec, **kwargs):
```

Comment:
yes. This has the unfortunate drawback that you actually have to initialize the view when you generate the documentation, which might have some undesired side effect